### PR TITLE
ComputeNode: Fix usage (much more particles)

### PR DESCRIPTION
**Pull request by sunag**
Related issue: https://github.com/mrdoob/three.js/pull/23905

**Description**

The previous approach does not multiply the number of elements by `workgroupSize`, now we can compute huge arrays.
I updated `webgpu_compute` example and change the behavior of `ComputeNode`.

`webgpu_compute` now work with, 300.000 particles.

--
I created a small internal example computing vertices(position) above 16-bits and everything worked ok with `ComputeNode`.

https://user-images.githubusercontent.com/502810/165180206-8caeaaee-6a11-4627-b365-2f247102f70c.mp4

*This contribution is funded by [Google via Igalia](https://igalia.com)*

### CODE REVIEW

File: examples/webgpu_compute.html [hunk 1]
```diff
@@ -116,19 +116,24 @@
...
+				pointsGeometry.setAttribute( 'particle', particleBuffer ); // dummy the position points as instances
+				pointsGeometry.drawRange.count = 1; // force render points as instances ( not triangle )
 
 				const pointsMaterial = new Nodes.PointsNodeMaterial();
-				pointsMaterial.colorNode = add( positionLocal, color( 0xFFFFFF ) );
+				pointsMaterial.colorNode = add( particleNode, color( 0xFFFFFF ) );
+				pointsMaterial.positionNode = particleNode;
 
 				const mesh = new THREE.Points( pointsGeometry, pointsMaterial );
+				mesh.isInstancedMesh = true;
```
Comment (mrdoob) [inline examples/webgpu_compute.html:135]: Hmmm... Someday we'll have to figure out how to make a better abstraction for instancing...

**Comment by mrdoob** [comment]
Thanks!

File: examples/webgpu_compute.html [hunk 2]
```diff
@@ -116,19 +116,24 @@
...
+				mesh.count = particleNum;
```
Comment (mrdoob) [inline examples/webgpu_compute.html:136]: Hmmm... Someday we'll have to figure out how to make a better abstraction for instancing...

Comment (sunag) [reply examples/webgpu_compute.html:136]: I think in something like this: https://github.com/mrdoob/three.js/pull/23835#discussion_r844829963

```js
const instancedObj3d = new InstanceObject3D( mesh || sprite3d || skinnedMesh, instanceCount );
instancedObj3d.target; // mesh || sprite3d || skinnedMesh
instancedObj3d.count; // instanceCount 

scene.add( instancedObj3d  );
```

**Pull request merged**